spec/models/user_spec.rb

```diff
@@ -1,4 +1,133 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
+  subject { described_class.new }
+
+  it 'user is valid' do
+    subject.first_name = 'John'
+    subject.last_name = 'Doe'
+    subject.email = 'johndoe@gmail.com'
+    subject.username = 'jdoe'
+    subject.phone_number = '0449 774 759'
+
+    expect(subject).to be_valid
+  end
+
+  # Verifying if object attribute presence is functional
+  it 'first name is present' do
+    subject.first_name = ' '
+    expect(subject).to_not be_valid
```

Comment:
(style) prefer `.not_to` over `.to_not`

Reply:
Will do! Definitely going to investigate the lib functions provided for rspec soon.